Выполнила ДЗ№2

FilosofiaJulia opened the pull request:
Выделила основные компоненты, проект собирается, запускается. Вроде все работает.

FilosofiaJulia committed:
Moved the project from the first module

создала компоненты: header, footer, navigation, popup, banner, breadcrumbs,  partners, элементы из блога. Не смогла выделить кнопки/ссылки и Category Products,  так как не разобралась, как добавить им классы(стили) для разных страниц

FilosofiaJulia committed:
added the button component, fixed styles

Yan1993-Dev commented on source/index.html:
```diff
@@ -22,19 +22,105 @@
+            <li class="products__item item_1">
+                <img class="products__item-img" src="./img/apparel-kids.png" alt="apparel kids" width="280" height="270">
+                <a class="products__link" href="#">APPAREL KIDS</a>
+            </li>
```
Можно вынести в отдельный компонент

Yan1993-Dev commented on source/index.html:
```diff
@@ -22,19 +22,105 @@
+    </section>
+    <section class="container partners">
+        <h2 class="visually-hidden">Partnes</h2>
+        @@include('blocks/partners.html',
```
Отлично, так и нужно

Yan1993-Dev commented on source/index.html:
```diff
@@ -22,19 +22,105 @@
+    <div class="container two-column">
+        <section class="about">
+            <h2 class="about__title">Design Life Kids</h2>
+            <a class="about__link" href="#">About Us</a>
```
Ссылки можно тоже вынести в отдельные компоненты

Yan1993-Dev commented on source/index.html:
```diff
@@ -22,19 +22,105 @@
+            <p class="about__info">Some days i feel like i've been working on dlk my entire life
+                and there are other days that i am completely shocked 
+                that four years have flown by. It's simply magical.</p>
+            <a class="button buy-link" href="/catalog.html">buy now</a>
```
И это тоже будет компонентом. Логику думаю ты поняла

FilosofiaJulia committed:
Moved the file with the rules for fonts

FilosofiaJulia committed:
fixed styles for buttons

FilosofiaJulia committed:
fixed header

Изменила название классов по БЭМ, сделала пункты меню компонентом, изменила стили по правилам scss

FilosofiaJulia committed:
fixed the block with product categories

Изменила название классов по БЭМ, сделала карточки категорий продуктов компонентом, исправила их стили для главной страницы и каталога

FilosofiaJulia committed:
fixed classes and styles on the main page

Изменила название классов по БЭМ, сделала ссылки с общими стилями компонентом, переписала стили в scss

FilosofiaJulia committed:
fixed banner

Изменила название классов по БЭМ, стили по правилам scss

FilosofiaJulia committed:
fixed footer

Изменила название классов по БЭМ, сделала ссылки компонентом, изменила стили по правилам scss

FilosofiaJulia committed:
fixed breadcrumbs and navigation

Изменила название классов по БЭМ, стили по правилам scss

FilosofiaJulia committed:
fixed contact, toys pages

Переименовала классы по БЭМ, изменила некоторые стили, сделала карточки на странице toys компонентом

FilosofiaJulia committed:
fixed styles

создала переменные для основных цветов и шрифтов

FilosofiaJulia committed:
add mixin

FilosofiaJulia committed:
moved the form to components

FilosofiaJulia commented:
Выполнила ДЗ№3
Изменила название классов по БЭМ. 
Изменила стили, используя вложенности, задала основные цвета и шрифты переменными, поэкспериментировала с миксином и расширением.

Пожалуйста, обрати внимание на комментарий в компоненте filter. 
Сделала форму компонентом, но остается проблема с плейсхолдером (есть комментарий).

Yan1993-Dev commented on source/scss/_base.scss:
```diff
@@ -1,23 +1,64 @@
-  -webkit-tap-highlight-color: transparent;
+* {
+  margin: 0;
+  padding: 0;
```
box-sizing: border-box; зачем убрала?

FilosofiaJulia replied:
Убрала еще в первом модуле. Ломал мне верстку в банере.

FilosofiaJulia committed:
fixed filter styles

FilosofiaJulia committed:
fixed checkbox styles

Проблема с плейсхолдером

FilosofiaJulia committed:
add an agreement monitoring function

FilosofiaJulia committed:
add transition to elements with different states

FilosofiaJulia committed:
add transition to elements with different states

FilosofiaJulia committed:
fixed styles

FilosofiaJulia committed:
add animation scene

Анимация работает нормально только при ширине экрана 1400

FilosofiaJulia commented:
Выполнила ДЗ№4
Анимация работает отлично только при ширине экрана 1400 px. Если выставлять меньше, то облачка обрезаются, и анимация такси ломается.

FilosofiaJulia committed:
fixed styles animation pages

Сделала анимацию бесконечной

FilosofiaJulia committed:
fixed styles animation page

FilosofiaJulia committed:
add wheels spin animation

FilosofiaJulia committed:
add svg animation page

анимации запускаются и выключаются по клику на кнопки

FilosofiaJulia commented:
Выполнила ДЗ№5
Анимации работают. Надо довести до ума функции, так как анимации запускаются и выключаются при нажатии на кнопки.

FilosofiaJulia committed:
change class name

FilosofiaJulia committed:
fixed animation Desktop

FilosofiaJulia commented:
Решена проблема анимации.

FilosofiaJulia committed:
add breakpoints for header

FilosofiaJulia committed:
add burger menu

FilosofiaJulia committed:
fixed burger menu

FilosofiaJulia committed:
add adaptive main page without banner and services

FilosofiaJulia committed:
fixed main page styles

FilosofiaJulia committed:
add adaptive popup, contact page

FilosofiaJulia committed:
add adaptive blog page

FilosofiaJulia committed:
add adaptive catalog page

FilosofiaJulia committed:
add adaptive toys page

FilosofiaJulia committed:
add adaptive svg page

FilosofiaJulia committed:
change animation page styles

FilosofiaJulia committed:
add fonts and images for grid page

FilosofiaJulia committed:
add grid page

FilosofiaJulia committed:
add adaptive grid page

FilosofiaJulia committed:
change headings styles

FilosofiaJulia committed:
add adaptive animation page

FilosofiaJulia committed:
change main page styles

FilosofiaJulia commented:
Выполнила ДЗ№6 и ДЗ№7 + сверстала адаптивную страницу Grid.  Все страницы из макета адаптивны. Исправила ошибку в предполагаемом слайдере в разделе "Категории продуктов" на главной странице.

FilosofiaJulia committed:
change fonts styles

FilosofiaJulia committed:
create bootstrap page

FilosofiaJulia committed:
make the audio player a component

FilosofiaJulia committed:
change styles of grid page

FilosofiaJulia committed:
add SEO settings

FilosofiaJulia committed:
Update starry-sky.png

FilosofiaJulia committed:
fix adaptive layout bugs

FilosofiaJulia committed:
fix header bug
